[
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/2170838",
    "html_url": "https://github.com/ipython/ipython/issues/816#issuecomment-2170838",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/816",
    "id": 2170838,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxNzA4Mzg=",
    "user": {
      "login": "minrk",
      "id": 151929,
      "node_id": "MDQ6VXNlcjE1MTkyOQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/151929?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/minrk",
      "html_url": "https://github.com/minrk",
      "followers_url": "https://api.github.com/users/minrk/followers",
      "following_url": "https://api.github.com/users/minrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/minrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/minrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/minrk/subscriptions",
      "organizations_url": "https://api.github.com/users/minrk/orgs",
      "repos_url": "https://api.github.com/users/minrk/repos",
      "events_url": "https://api.github.com/users/minrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/minrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2011-09-22T18:50:36Z",
    "updated_at": "2011-09-22T18:50:36Z",
    "author_association": "MEMBER",
    "body": "dateutil 2.0 is for Python >= 3.0 only.\n\nFrom the [dateutil homepage](http://labix.org/python-dateutil#line-12):\n\n> If you need dateutil for Python 2.X, please continue using the 1.X series.\n"
  },
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/2171024",
    "html_url": "https://github.com/ipython/ipython/issues/816#issuecomment-2171024",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/816",
    "id": 2171024,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxNzEwMjQ=",
    "user": {
      "login": "dartdog",
      "id": 87834,
      "node_id": "MDQ6VXNlcjg3ODM0",
      "avatar_url": "https://avatars2.githubusercontent.com/u/87834?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/dartdog",
      "html_url": "https://github.com/dartdog",
      "followers_url": "https://api.github.com/users/dartdog/followers",
      "following_url": "https://api.github.com/users/dartdog/following{/other_user}",
      "gists_url": "https://api.github.com/users/dartdog/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/dartdog/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/dartdog/subscriptions",
      "organizations_url": "https://api.github.com/users/dartdog/orgs",
      "repos_url": "https://api.github.com/users/dartdog/repos",
      "events_url": "https://api.github.com/users/dartdog/events{/privacy}",
      "received_events_url": "https://api.github.com/users/dartdog/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2011-09-22T19:07:19Z",
    "updated_at": "2011-09-22T19:07:19Z",
    "author_association": "NONE",
    "body": "Yup,, found it.. dunno why the default easy_install install gave me 2.0.. Which I took... Sorry.\n"
  },
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/2171196",
    "html_url": "https://github.com/ipython/ipython/issues/816#issuecomment-2171196",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/816",
    "id": 2171196,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxNzExOTY=",
    "user": {
      "login": "minrk",
      "id": 151929,
      "node_id": "MDQ6VXNlcjE1MTkyOQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/151929?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/minrk",
      "html_url": "https://github.com/minrk",
      "followers_url": "https://api.github.com/users/minrk/followers",
      "following_url": "https://api.github.com/users/minrk/following{/other_user}",
      "gists_url": "https://api.github.com/users/minrk/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/minrk/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/minrk/subscriptions",
      "organizations_url": "https://api.github.com/users/minrk/orgs",
      "repos_url": "https://api.github.com/users/minrk/repos",
      "events_url": "https://api.github.com/users/minrk/events{/privacy}",
      "received_events_url": "https://api.github.com/users/minrk/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2011-09-22T19:24:04Z",
    "updated_at": "2011-09-22T19:24:04Z",
    "author_association": "MEMBER",
    "body": "yeah, that's a problem.  PyPI only lists 1.5, but setuptools traverses to the home page and/or download page, where it finds 2.0.\n\nThey need to fix their listing, to prevent setuptools from finding 2.0.\n"
  }
]
